recommend/views.py

```diff
@@ -96,4 +96,15 @@ def api_recommendations_team(request, id):
 
 class DetailSuggestedRecommendations(generics.RetrieveUpdateDestroyAPIView):
     queryset = SuggestedRecommendation.objects.all()
-    serializer_class = SuggestedRecommendationSerializer
\ No newline at end of file
+    serializer_class = SuggestedRecommendationSerializer
+
+@api_view(['GET'])
+def api_dora_kpi(request, id):
```

Comment:
```suggestion
def api_dora_kpi(request, id, measurement_id):
```

Reply:
So that specific measurements in the past can also be queried 
-> and in urls.py add a url pattern with <int: id> and <int: measurement_id>

Alternatively instead of measurement_id we could also use year, month, day to make it more intuitive
-> ask product owner?

Reply:
If we were to use y, m, d, the uniqueness would be ensured by the data in the request, right?

Reply:
Ja weil theoretisch bekommen wir keine doppelte Measurements an einem Tag für ein Team